### ISSUE TITLE
Failing test /tests/plugins/autocomplete/manual/a11ydivarea

### ISSUE THREAD

**Issue by FilipTokarski** [ckeditor/ckeditor4#4707]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

Open /tests/plugins/autocomplete/manual/a11ydivarea test on Safari.

### Expected result

VoiceOver reads dropdown items when autocomplete is triggered

### Actual result

VoiceOver fails to read dropdown items with autocomplete

## Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by f1ames** [comment]
Could be a follow-up of #4617. I would say this is a feature request since [VoiceOver is not officially supported](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_browsers.html#accessibility-support).
